src/crudtask.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-alert */
+// eslint-disable-next-line import/no-named-default
+import checkboxesEvent, { list, save } from './status-update.js';
+
+
+const todoList = document.getElementById('todo-list');
+
+
+export function edit() {
+  const edit = document.querySelectorAll('span');
+  for (let i = 0; i < edit.length; i += 1) {
+    edit[i].addEventListener('click', () => {
+      if (true) {
+        const { id } = edit[i].parentNode;
+        if (edit[i].textContent) {
+          edit[i].addEventListener('mouseleave', (event) => {
+            localStorage.setItem('edit', JSON.stringify(edit[i].textContent));
+            list[i].description = JSON.parse(localStorage.getItem('edit'));
+            save();
+            displayTasks();
+          });
+        }
+      }
+    });
+  }
+}
+
+export function clear() {
+  document.getElementById('clear-complete').addEventListener('click', (event) => {
+    const filtered = list.filter((element) => {
+      if (element.completed === false) {
+        return true;
+      }
+      return false;
+    });
+    list = filtered;
+    for (let i = 0; i < list.length; i += 1) {
+      list[i].index = i;
+    }
+    checkboxesEvent();
+    save();
+  });
+}
```

Comment:
- [ ] Good job, your is not working properly as expected. when the checkbox is checked and I clicked on the clear all completed button then it should remove it from the list but it's not working that, please look into this and fix it